test/blocks/quiz/mocks/index.html

```diff
@@ -0,0 +1,25 @@
+<div>
+  <div class="quiz">
+    <div>
+      <div data-align="center" data-valign="middle">quizUrl</div>
```

Comment:
This is the raw data coming straight out from the doc to html conversion before our JS kicks in. I don't think Cody has changed anything on that part. Lemme know if you think otherwise, I will change it in the upcoming main PR to main then.

Reply:
@sabyamon No changes from me at all on the quiz block config stuff - just the markup for the quiz that comes out of preact.